internal/plugins/helm/v1/scaffolds/init.go

```diff
@@ -32,7 +32,7 @@ import (
 
 const (
 	// kustomizeVersion is the sigs.k8s.io/kustomize version to be used in the project
-	kustomizeVersion = "v3.8.7"
+	kustomizeVersion = "v4.5.2"
```

Comment:
We cannot upgrade the kustomize version here because it is breaking change for the stable plugins. 
/hold